src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Everyone disperse, the truck started work!!!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Having accepted the job, the truck finished loading !");
```

Comment:
Similarly, the `stopWork` method should provide an informative message that includes the type of machine, 'Truck', that has stopped working. The current message could be improved for clarity and specificity.